db/schema/06_photos.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS photos CASCADE;
+
+CREATE TABLE photos (
+  id SERIAL PRIMARY KEY NOT NULL,
+  photo_url VARCHAR(255) NOT NULL,
+  category VARCHAR(255) NOT NULL
```

Comment:
Can we make this nullable in case we want to allow users to insert their own photos in the future?